src/Domain/LabelMonetaryAccount.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types = 1);
+
+namespace Link0\Bunq\Domain;
+
+final class LabelMonetaryAccount
+{
+    /**
+     * @var string
+     */
+    protected $iban;
```

Comment:
Could you make these properties private? I believe in closing down a class as much as possible, because opening it up later when needed is much more easy than closing it down when others are potentially (ab)using it.